core/src/rules/optimising_line_formatter/contexts.rs

```diff
@@ -604,6 +611,21 @@ impl<'a> SpecificContextStack<'a> {
             (Some(TT::Keyword(KK::Type)), Some(tt)) if tt != TT::Keyword(KK::Of) => {
                 self.update_last_matching_context(node, context_matches!(_), apply_pivotal_break);
             }
+            (Some(TT::Keyword(KK::Of)), Some(TT::Op(OK::LParen))) => {
+                /*
+                    To exclude `set of` anonymous enum, e.g.,
+                    ```delphi
+                    type
+                      A = set of (
+                          a,
+                          b
+                      );
+                    ```
+                */
+                self.update_last_matching_context(node, context_matches!(_), |_, data| {
+                    data.is_broken |= is_break;
+                });
+            }
```

Comment:
The code inside this branch is technically redundant, because in this case the requirement is always `MustNotBreak`.
Would it be incorrect to use `apply_privotal_break` here? Just to make it shorter.

Also, I don't think the comment needs to be so detailed with an example.

Reply:
Redundant, I would say yes - to which I say we can remove it.
We definitely can't use `apply_pivotal_break`, because that would invalidate the case we are covering by saying you can break the comma list if you don't break the `(`.

Overall, I can remove the example from the comment and remove code. Leaving this?
```suggestion
            (Some(TT::Keyword(KK::Of)), Some(TT::Op(OK::LParen))) => {
                // to exclude `set of` anonymous enum
            }
```

What do you think?

Reply:
Much better